core/src/main/resources/jenkins/security/ResourceDomainConfiguration/help-resourceRootUrl.html

```diff
@@ -0,0 +1,40 @@
+<p>
```

Comment:
Might be a good idea to add an example URL somewhere near the top?

Reply:
Not sure. There are no "good" example URLs that make sense and can easily be used, unless we go full fantasy-land with `builds.acme.org` and `build-content.acme.org`. Or is this useful enough?

Suggestions?

Reply:
Significantly improved form validation makes this less important.